[2021-05-08 01:12:06,256] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: example_python_operator.virtualenv_python 2021-05-07T19:41:47.213625+00:00 [queued]>
[2021-05-08 01:12:06,272] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: example_python_operator.virtualenv_python 2021-05-07T19:41:47.213625+00:00 [queued]>
[2021-05-08 01:12:06,272] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-08 01:12:06,273] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2021-05-08 01:12:06,273] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-08 01:12:06,384] {taskinstance.py:1089} INFO - Executing <Task(PythonVirtualenvOperator): virtualenv_python> on 2021-05-07T19:41:47.213625+00:00
[2021-05-08 01:12:06,388] {standard_task_runner.py:52} INFO - Started process 444739 to run task
[2021-05-08 01:12:06,397] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_python_operator', 'virtualenv_python', '2021-05-07T19:41:47.213625+00:00', '--job-id', '3', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/example_dags/example_python_operator.py', '--cfg-path', '/tmp/tmpao1axrue', '--error-file', '/tmp/tmppnc2t510']
[2021-05-08 01:12:06,398] {standard_task_runner.py:77} INFO - Job 3: Subtask virtualenv_python
[2021-05-08 01:12:06,656] {logging_mixin.py:104} INFO - Running <TaskInstance: example_python_operator.virtualenv_python 2021-05-07T19:41:47.213625+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-08 01:12:06,782] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_python_operator
AIRFLOW_CTX_TASK_ID=virtualenv_python
AIRFLOW_CTX_EXECUTION_DATE=2021-05-07T19:41:47.213625+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-07T19:41:47.213625+00:00
[2021-05-08 01:12:06,784] {process_utils.py:135} INFO - Executing cmd: virtualenv /tmp/venv6_3g71bc
[2021-05-08 01:12:06,799] {process_utils.py:137} INFO - Output:
[2021-05-08 01:12:10,797] {process_utils.py:141} INFO - created virtual environment CPython3.8.3.final.0-64 in 3523ms
[2021-05-08 01:12:10,798] {process_utils.py:141} INFO -   creator CPython3Posix(dest=/tmp/venv6_3g71bc, clear=False, global=False)
[2021-05-08 01:12:10,798] {process_utils.py:141} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/navneetsajwan/.local/share/virtualenv)
[2021-05-08 01:12:10,799] {process_utils.py:141} INFO -     added seed packages: pip==20.2.4, setuptools==50.3.2, wheel==0.35.1
[2021-05-08 01:12:10,799] {process_utils.py:141} INFO -   activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
[2021-05-08 01:12:10,927] {process_utils.py:135} INFO - Executing cmd: /tmp/venv6_3g71bc/bin/pip install colorama==0.4.0
[2021-05-08 01:12:10,936] {process_utils.py:137} INFO - Output:
[2021-05-08 01:12:21,941] {process_utils.py:141} INFO - Collecting colorama==0.4.0
[2021-05-08 01:12:23,976] {process_utils.py:141} INFO -   Downloading colorama-0.4.0-py2.py3-none-any.whl (21 kB)
[2021-05-08 01:12:24,534] {process_utils.py:141} INFO - Installing collected packages: colorama
[2021-05-08 01:12:24,862] {process_utils.py:141} INFO - Successfully installed colorama-0.4.0
[2021-05-08 01:12:26,207] {process_utils.py:141} INFO - WARNING: You are using pip version 20.2.4; however, version 21.1.1 is available.
[2021-05-08 01:12:26,207] {process_utils.py:141} INFO - You should consider upgrading via the '/tmp/venv6_3g71bc/bin/python -m pip install --upgrade pip' command.
[2021-05-08 01:12:26,429] {process_utils.py:135} INFO - Executing cmd: /tmp/venv6_3g71bc/bin/python /tmp/venv6_3g71bc/script.py /tmp/venv6_3g71bc/script.in /tmp/venv6_3g71bc/script.out /tmp/venv6_3g71bc/string_args.txt
[2021-05-08 01:12:26,449] {process_utils.py:137} INFO - Output:
[2021-05-08 01:12:26,521] {process_utils.py:141} INFO - [31msome red text
[2021-05-08 01:12:26,521] {process_utils.py:141} INFO - [42mand with a green background
[2021-05-08 01:12:26,522] {process_utils.py:141} INFO - [2mand in dim text
[2021-05-08 01:12:26,522] {process_utils.py:141} INFO - [0m
[2021-05-08 01:12:26,522] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:12:36,531] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:12:46,539] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:12:56,549] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:13:06,558] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:13:16,567] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:13:26,584] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:13:36,591] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:13:46,598] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:13:56,606] {process_utils.py:141} INFO - [2mPlease wait...
[2021-05-08 01:14:06,616] {process_utils.py:141} INFO - Finished
[2021-05-08 01:14:06,689] {python.py:118} INFO - Done. Returned value was: None
[2021-05-08 01:14:06,696] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=example_python_operator, task_id=virtualenv_python, execution_date=20210507T194147, start_date=20210507T194206, end_date=20210507T194406
[2021-05-08 01:14:06,843] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-08 01:14:06,878] {local_task_job.py:146} INFO - Task exited with return code 0
